app/autotranslate/server/settings.js

```diff
@@ -54,4 +57,18 @@ Meteor.startup(function() {
 				_id: 'AutoTranslate_ServiceProvider', value: 'deepl-translate',
 			}],
 	});
+
+	settings.add('AutoTranslate_MicrosoftAPIKey', '', {
+		type: 'string',
+		group: 'Message',
+		section: 'AutoTranslate_Microsoft',
+		public: true,
```

Comment:
Should not be public since it's not used on the client side

Reply:
Applies to the other APIKeys too!

Reply:
Changed all API-Keys  setting in translation context to `private`